packages/github/src/context.ts

```diff
@@ -21,7 +20,9 @@ export class Context {
    */
   constructor() {
     this.payload = process.env.GITHUB_EVENT_PATH
-      ? require(process.env.GITHUB_EVENT_PATH)
+      ? JSON.parse(
+          readFileSync(process.env.GITHUB_EVENT_PATH, {encoding: 'utf8'})
```

Comment:
Do we want to add error handling here? Make sure the file exists first and print message if not?

Reply:
And while we are here, should we debug print a serialization of the context?

Reply:
Error handling yes, serialization of the context seems like a no to me. Its pretty massive, and since we're built to be a dependency I'm a little leary of large debug output that may or may not be helpful, especially since lots of actions probably won't use that directly.